units/sdlkeycode.inc

```diff
@@ -0,0 +1,318 @@
+//from "sdl_keycode.h"
+
+  {**
+   *  The SDL virtual key representation.
+   *
+   *  Values of this type are used to represent keyboard keys using the current
+   *  layout of the keyboard.  These values include Unicode values representing
+   *  the unmodified character that would be generated by pressing the key, or
+   *  an SDLK_* constant for those keys that do not generate characters.
+   *
+   *  A special exception is the number keys at the top of the keyboard which
+   *  always map to SDLK_0...SDLK_9, regardless of layout.
+   *}
+type
+  PSDL_KeyCode = ^TSDL_KeyCode;
+  TSDL_KeyCode = type cint32;
+
+const
+  SDLK_SCANCODE_MASK = 1 shl 30;
+  //#define SDL_SCANCODE_TO_KEYCODE(X)  (X | SDLK_SCANCODE_MASK)
```

Comment:
I prefer to keep it as is, because a function call has a little more overhead compared to just reading a constant value. Since scancodes may be requested in a large amount and rapidly, this means a large amount of function calls would result. 

A second point why I opt'ed for keeping it as is, it saved some time :-). 

Feel free to merge this if you approve my answer.